### ISSUE TITLE
'origen new dut' doesn't generate code that runs dut level initialize

### ISSUE THREAD

**Issue by xnappo** [Origen-SDK/origen#401]
origen new dut creates a directory structure with two model.rb files, one at the 'dut' level, and one at the 'derivative' level.  both contain an initialize.

as generated, the initialize at the 'dut' level does not get called.

for ease of use, recommend adding a 'super' in the derivative level initialize by default with a comment regarding why one might want to disable it.

**Comment by priyavadan** [comment]
@xnappo the change probably needs to be [here](https://github.com/Origen-SDK/origen/blob/master/templates/code_generators/model.rb#L11)

pseudo code:

~~~ruby
<% if @root_class -%>
<% if @top_level -%>
  include Origen::TopLevel
<% else -%>
  include Origen::Model
<% end -%>

  def initialize(options = {})
  end
<% else -%>
  def initialize(options = {})
    super
  end
<% end -%>
~~~

I have pushed a branch [here](https://github.com/Origen-SDK/origen/commit/8e927d6b278318eed0c3765ccc6b32c85691a16c) with the code, but can you please verify :)

**Comment by xnappo** [comment]
Dang Origen support is fast :D We were having an email convo about this and I was just logging this here until I got a chance to do it - thanks!

Tested and works as intended - but should we add a comment about what it does and why one might want to comment it out?

**Comment by priyavadan** [comment]
@xnappo yes, do you mind adding in the comment. I can too, later today if needed.

I ran into it myself and saw your issue log pop up so thought will just update it too!

**Comment by priyavadan** [comment]
fixed in v0.59.2
